services/exposurelog/values.yaml

```diff
@@ -20,6 +20,17 @@ image:
   # @default -- The appVersion of the chart
   tag: ""
 
+# -- Postgres database connection information
```

Comment:
You want to omit the comment here since right now it's documenting both `db` and all its child settings.